satpy/tests/reader_tests/test_ahi_hsd.py

```diff
@@ -140,7 +140,11 @@ def test_region(self, fromfile, np2str):
                             "spare": ""}
 
             area_def = fh.get_area_def(None)
-            proj_dict = area_def.proj_dict
+            with warnings.catch_warnings():
+                warnings.filterwarnings("ignore",
+                                        message=r"You will likely lose important projection information",
+                                        category=UserWarning)
+                proj_dict = area_def.proj_dict
```

Comment:
It seems I could get all the tested things from `area_def.crs.to_cf()` apart from the `units`. There is `LENGTHUNIT["metre",...]` in `crs_wkt`, but checking for that seems like a cludge. So it's either using `proj_dict` or doing `"metre" in area_def.crs.to_cf()['crs_wkt']` :thinking:

Reply:
Maybe `crs.axis_info[0].unit_name`?

Reply:
But how about creating an expected crs instead, and comparing it to area_def.crs?
eg:
```python
expected_crs = CRS.from_dict(dict(a=6378137.0, b=6356752.3, h=35785863.0, lon_0=140.7, proj="geos", units="m"))
assert area_def.crs == expected_crs
```

Reply:
Too simple :see_no_evil: Adjusted in https://github.com/pytroll/satpy/pull/2689/commits/24422ad5ea9e59e41b1daac50d6842e7d8f0b9ab

Reply:
Ok, I didn't comment on the other ones, but there are a bunch other places where this could be used instead of suppressing the warning...

Reply:
I tried to go the "do not suppress" route, but evidently I missed this way. I'll start combing through the changes for these.

Reply:
I updated the `proj_dict` usages to `CRS` as much as possible. The remaining are used either because the original projection needs to be modified (in `nwcsaf_nc`) and the dict is just too simple not to be used, or because the CRS objects differ even if the proj dicts are identical (`oceancolorcci_l3_nc`).